src/runtime/swg-client-event-manager.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2019 The Subscribe with Google Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as EventApi from '../api/swg-client-event-manager-api';
+
+/** @implements {EventApi.SwgClientEventManagerApi}*/
+export class SwgClientEventManager {
+  constructor() {
+  }
+
+  /**
+   * Ensures the callback function is notified anytime one of the passed
+   * events occurs unless a filterer returns false.
+   * @param {!function(!EventApi.SwgClientEvent)} unusedCallback
+   * @overrides
```

Comment:
I think it's just `@override`. Also, I usually prefer to skip the docs (and often args) to avoid documentation propagating in all implementations. `@override` basically directs the reader "see docs at the interface".